site-src/references/glossary/index.md

```diff
@@ -0,0 +1,20 @@
+# Glossary
+
+About API: A cluster-local CRD to store arbitrary properties about a cluster. See [About API Overview](concepts/about-api.md). _Previously known as ClusterProperty API._
+
+Cluster Inventory: An old name for the project that became the ClusterProfile API. See _ClusterProfile API_.
+
+ClusterProfile API: A CRD intended to store information about member clusters in a clusterset. See [ClusterProfile API Overview](concepts/clusterprofile-api.md). _Previously known as Cluster Inventory._
+
+ClusterProperty API: An old name for the About API. See also _About API_.
```

Comment:
Is this name really deprecated considering that the CRD is named ClusterProperty still? 🤔

Reply:
No big deal I guess, since we refer to the `Above API` above. :)

Reply:
Removed in [5ca42a6](https://github.com/kubernetes-sigs/sig-multicluster-site/pull/41/commits/5ca42a6f9f0f6b62abea7afc09c705c1be9c95e8), along with other old names and aliases, if we want them later I will add them later.